python-stream/send.py

```diff
@@ -14,14 +14,15 @@ async def send():
         password="guest",
     ) as producer:
         await producer.create_stream(
-            STREAM_NAME, exists_ok=True, arguments={"MaxLengthBytes": STREAM_RETENTION}
+            STREAM_NAME, exists_ok=True, arguments={"max-length-bytes": STREAM_RETENTION}
         )
 
         await producer.send(stream=STREAM_NAME, message=b"Hello, World!")
 
         print(" [x] Hello, World! message sent")
 
         input(" [x] Press Enter to close the producer  ...")
+        await producer.close()
```

Comment:
In the rstream library the producer.close is called directly in the __aexit__ method. 
As in the example above we are using the AsyncContext this will be automatically called when 
we exit from it https://github.com/qweeze/rstream/blob/master/rstream/producer.py#L142 (same for Consumer)

Reply:
removed in 92134198a760bf6dc18fde5a57694ec6e4c4be28, thank you!